diskplorer.py

```diff
@@ -148,17 +157,18 @@ def fix_y_axis(plt):
 
 ax2 = ax1.twinx()
 #ax2.plot(concurrencies, latencies, 'r-+')
-ax2.errorbar(concurrencies, latencies, yerr=[latencies_05, latencies_95], color='r')
-ax2.set_ylabel(u'average latency (ns)', color='r')
+ax2.errorbar(concurrencies, latencies, yerr=[latencies_05, latencies_99], color='r')
```

Comment:
Why did you need to switch from 05-95 to 05-99.
Let's keep the original 05-95th as min-max by default and we can introduce a parameter that would allow choosing from other options (enum) for both left and right sides of the range.

Reply:
creating a commit that reverts to 95p